tools/cldr-apps/src/main/java/org/unicode/cldr/web/SurveyMain.java

```diff
@@ -145,30 +143,14 @@ public enum Phase {
          * TC/Admin may make changes, but the SurveyTool is locked to any lower levels.
          */
         VETTING_CLOSED("Vetting Closed", CheckCLDR.Phase.FINAL_TESTING),
-        /**
-         * DEPRECATED: use VETTING_CLOSED
-         *
-         * @see #VETTING_CLOSED
-         * @deprecated
-         */
-        @Deprecated
-        CLOSED("Closed", CheckCLDR.Phase.FINAL_TESTING),
-        /**
-         * DEPRECATED: Not to be used in the SurveyTool.
-         *
-         * @see #VETTING_CLOSED
-         * @deprecated
-         */
-        @Deprecated
-        FINAL_TESTING("Final Testing", CheckCLDR.Phase.FINAL_TESTING),
+
         /** The SurveyTool is not open for any changes. */
         READONLY("Read-Only", CheckCLDR.Phase.FINAL_TESTING),
+
         /**
-         * Do not use.
-         *
-         * @deprecated
+         * Survey Tool is in Beta mode. Votes, announcements, etc., will be stored in special
+         * database tables with the "_beta" suffix, and will be used only for testing.
          */
-        @Deprecated
         BETA("Beta", CheckCLDR.Phase.SUBMISSION);
```

Comment:
I started to delete BETA along with the other deprecated items, then noticed that we did use BETA at the start of v44, and there's a lot of dependent code on both the front and back ends. Unlike the other deprecated items, which have caused a lot of confusion due to their overlapping/redundant meanings, BETA seems like it may still be useful and isn't likely to be used accidentally. It shouldn't be deprecated unless we actually intend to get remove it.